src/google/protobuf/stubs/stringpiece_unittest.cc

```diff
@@ -707,17 +707,17 @@ TEST(StringPiece, Contains) {
   EXPECT_TRUE(!a.contains(d));
 }
 
-TEST(StringPiece, NULLInput) {
+TEST(StringPiece, nullptrInput) {
```

Comment:
How about "NullInput" just to make the test case name go with our naming convention.

Reply:
Ack